Daily_Reward/DailyReward.txt

```diff
@@ -42,19 +43,24 @@ OnWhisperGlobal:
 OnLoginCmnd:
 OnPCLoginEvent:
 	if(.IPCheck){
-		query_sql("SELECT last_ip FROM `login` WHERE account_id = "+getcharid(3)+"", .@LastIp$);
-		query_sql("SELECT account_id FROM `login` WHERE last_ip = '"+.@LastIp$+"'", .@AccountId);
-		if(getarraysize(.@AccountId) > .IPCheck){
-			message strcharinfo(0),"[Daily Rewards]: Rewards are limited to "+.IPCheck+" per IP sorry.";
+		query_sql("SELECT account_id FROM `login` WHERE last_ip = '"+getcharip()+"'", .@AccountId);
+		for(.@i=0;.@i<getarraysize(.@AccountId);.@i++){
+			query_sql("SELECT `value` FROM `acc_reg_num` WHERE `account_id` = '"+.@AccountId[.@i]+"' AND `key` = '#LastDailyReward'",.@LastIp2);
+			if(#LastDailyReward < .@LastIp2){
+				message strcharinfo(0),"[Daily Rewards]: Rewards are limited to 1 per IP sorry.";
 			end;
+			}
 		}
 	}
 	if(.MacCheck){
 		query_sql("SELECT last_unique_id FROM `login` WHERE account_id = "+getcharid(3)+"", .@last_unique_id$);
 		query_sql("SELECT account_id FROM `login` WHERE last_unique_id = '"+.@last_unique_id$+"'", .@AccountId2);
-		if(getarraysize(.@AccountId2) > .MacCheck){
-			message strcharinfo(0),"[Daily Rewards]: Rewards are limited to "+.MacCheck+" per computer sorry.";
+		for(.@i=0;.@i<getarraysize(.@AccountId2);.@i++){
+			query_sql("SELECT `value` FROM `acc_reg_num` WHERE `account_id` = '"+.@AccountId2[.@i]+"' AND `key` = '#LastDailyReward'",.@MacCheck2);
+			if(#LastDailyReward < .@MacCheck2){
```

Comment:
this check i think it's not working good with .MinWait
i think it need something like (#LastDailyReward + .MinWait - .@i) <.@MacCheck2
and moving .@i from down to up then change the message to say the time needed before getting the new reward
i am testing right now

Reply:
or saving the (#LastDailyReward or the max value from the other accounts) to a .@something after the gepard and ip check and changing the #LastDailyReward in the rest of the script to the new .@something ?